app/src/main/java/com/example/myapplication/mvp/view/CalculatorView.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.myapplication.mvp.view
+
+import android.app.Activity
+import com.example.myapplication.mvp.view.base.ActivityView
+import kotlinx.android.synthetic.main.activity_main.text_result
+import kotlinx.android.synthetic.main.activity_main.text_visor
+
+class CalculatorView(activity: Activity) : ActivityView(activity){
+
+    fun setVisor(operation : String){
+        activity!!.text_visor.text = operation
```

Comment:
Please, don´t use "!!" try to use .let  or elvis operator

Reply:
done